Build failure: wxPython fails inside `yt-dlg`

### Steps To Reproduce

Steps to reproduce the behavior:
1. update `nixos-unstable` from rev `9f4128e00b0ae8ec65918efeba59db998750ead6`
to `d0e1602ddde669d5beb01aec49d71a51937ed7be`
2. install/add/update `yt-dlg`

### Build log

```log
...
../../../../sip/cpp/sip_corewxArrayString.cpp: In function ‘int convertTo_wxArrayString(PyObject*, void**, int*, PyObject*)’:
../../../../sip/cpp/sip_corewxArrayString.cpp:91:26: error: ‘PyUnicode_GET_SIZE’ was not declared in this scope; did you mean ‘PyDict_GET_SIZE’?
   91 |             size_t len = PyUnicode_GET_SIZE(item);
      |                          ^~~~~~~~~~~~~~~~~~
      |                          PyDict_GET_SIZE
...
```
https://gist.github.com/GsakuL/7278d0a8dccd7ac77b8d3c1c7848954f

### Additional context

Add any other context about the problem here.

### Notify maintainers

@quantenzitrone 

### Metadata

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.6.36, NixOS, 24.11 (Vicuna), 24.11.20240824.d0e1602`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.18.5`
 - nixpkgs: `/nix/store/ia1zpg1s63v6b3vin3n7bxxjgcs51s2r-source`
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/yt/yt-dlg/package.nix
+++ b/pkgs/by-name/yt/yt-dlg/package.nix
@@ -1,27 +1,9 @@
 {
   lib,
-  python3,
+  python3Packages,
   fetchFromGitHub,
-  fetchPypi
+  fetchPypi,
 }:
-let
-  python3' =
-    (python3.override {
-      packageOverrides = final: prev: {
-        wxpython = prev.wxpython.overrideAttrs rec {
-          version = "4.2.0";
-          src = fetchPypi {
-            pname = "wxPython";
-            inherit version;
-            hash = "sha256-ZjzrxFCdfl0RNRiGX+J093+VQ0xdV7w4btWNZc7thsc=";
-          };
-        };
-      };
-    });
-
-  python3Packages = python3'.pkgs;
-
-in
 python3Packages.buildPythonApplication rec {
   pname = "yt-dlg";
   version = "1.8.5";
@@ -34,6 +16,7 @@ python3Packages.buildPythonApplication rec {
   };
 
   pyproject = true;
+  pythonRelaxDeps = [ "wxpython" ];
   build-system = with python3Packages; [
     setuptools
     wheel

